utp_internal.cpp

```diff
@@ -801,7 +802,7 @@ void UTPSocket::send_ack(bool synack)
 		// if the packet ack_nr + 1 has not yet
 		// been received
 		assert(inbuf.get(ack_nr + 1) == NULL);
-		size_t window = min<size_t>(14+16, inbuf.size());
+		size_t window = std::min<size_t>(14+16, inbuf.size());
```

Comment:
I know it's the pre-existing style but do we want to maybe reduce the number of explicitly specified template types and (safely) cast the arguments to be the same type instead? Could always do this later, or not at all, just thought I'd ask.

Reply:
I'll take a look. If there are places where it would be safer to let the compiler deduce the type, then maybe we should be doing that.

Reply:
@mikedld I've added some updates in 7480820 which removes the explicit template types in cases where the types were already in agreement.

There are still several places where IMO the intent was to transform the type, so I've left those explicitly-specified types alone.